src/components/ebay-breadcrumb/index.js

```diff
@@ -0,0 +1,43 @@
+const markoWidgets = require('marko-widgets');
+const emitAndFire = require('../../common/emit-and-fire');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+function getTemplateData(state, input) {
+    const htmlAttributes = processHtmlAttributes(input);
+    const inputItems = input.items || [];
+    let items = (inputItems).map((item, index) => {
+        const itemHtmlAttributes = processHtmlAttributes(item);
+        let tag = 'a';
+        let ariaCurrent = null;
+        const href = item.href || null;
+        const current = ((inputItems.length - 1) === index);
+        if (current && !href) {
+            tag = 'span';
+            ariaCurrent = 'page';
+        }
+        return {
+            tag,
+            htmlAttributes: itemHtmlAttributes,
+            renderBody: item.renderBody,
+            href,
+            ariaCurrent
+        };
+    });
+    items = Object.keys(items).length > 0 ? items : null;
+    return {
+        items,
+        headingText: input.headingText || '',
+        headingLevel: input.headingLevel || 'h2',
+        htmlAttributes: htmlAttributes
+    };
+}
+function handleClick(event, el) {
+    emitAndFire(this, 'breadcrumb-click', { event, currentTarget: el });
```

Comment:
`currentTarget` should ideally be `target`. The consumers probably need the anchor or span tag where the event originated and not the container `nav` element. They can still get it from the `event` object, but this will be easy.

Reply:
Emitting 3 values as `{ event, target: event.target, currentTarget: el }`

Reply:
Maybe we just need the `event` parameter and mention in docs that this event is fired from the root element. Developers can then use `event.detail.event.target` or `event.detail.event. currentTarget` based on their needs. Also renaming `event` to maybe `origEvent` will help to avoid confusion. But we need to check how other parts of the codebase is doing it.